### DISCUSSION TITLE
Totally New To GitHub! How to Use it efficiently ?

### DISCUSSION THREAD

**Discussion by amitkr0078** [community/community#22974] [New to GitHub]
<p>Hi There,</p>
<p>I just came across the GitHub and it’s an amazing place for developers. I also want to learn and contribute in GitHub. Can anybody tell me where to start. </p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/amitkr0078">@amitkr0078</a>,</p>
<p>Welcome to the Github Community Forum! And also to Github!</p>
<p>The absolutely first thing to do is complete the course: <a href="https://lab.github.com/githubtraining/introduction-to-github" rel="nofollow noopener">Introduction to Github</a>, <a href="https://lab.github.com/githubtraining/paths/first-day-on-github" rel="nofollow noopener">First Day on Github</a> and <a href="https://lab.github.com/githubtraining/paths/first-week-on-github" rel="nofollow noopener">First Week on Github</a>, you can find those course on the <a href="https://lab.github.com/" rel="nofollow noopener">github learning lab.</a></p>
<p>Then you probably want to contribute to a project but you don’t know how to do it and this is why exists the <a href="https://github.com/explore" rel="nofollow noopener">Github Explore.</a></p>
<p>If you want to help other people this is the community you ask for and for more informations you can read this post: <a href="https://github.community/t/getting-started-with-the-github-community-forum/48">Getting started with the GitHub Community Forum.</a></p>
<p>If you know Git, but not so much, you can read this awesome free book: <a href="https://git-scm.com/book/en/v2" rel="nofollow noopener">git pro v2.</a></p>
<p>If you have more question ask and the community will give you an answer :slight_smile:</p>
<p>-Gabriele-</p>
<p>Mark helpfull posts with<span> </span>Accept as Solution<span> </span>to help other users locate important info. Don’t forget to give<span> </span>Kudos<span> </span>for great contents!</p>
